genie-docs/src/docs/asciidoc/_properties.adoc

```diff
@@ -142,6 +142,106 @@ https://docs.spring.io/spring-boot/docs/current/reference/html/spring-boot-featu
 |false
 |no
 
+|genie.agent.launcher.titus.enabled
+|Whether the Titus agent launcher is active
+|false
+|no
+
+|genie.agent.launcher.titus.endpoint
+|The Titus API endpoint URI
+|https://example-titus-endpoint.tld:1234/api/v3/jobs
```

Comment:
should you bake the `/api/v3/jobs` part into the code and just let the URL be set since I assume the implementation depends on which version of the API it's using